packages/video_player/video_player/pubspec.yaml

```diff
@@ -4,6 +4,7 @@ description: Flutter plugin for displaying inline video with other Flutter
 repository: https://github.com/flutter/plugins/tree/main/packages/video_player/video_player
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+video_player%22
 version: 2.4.6
+publish_to: none
```

Comment:
Publishing should not be disabled for any of these packages.

Reply:
I did it to solve some warnings during the build. Once I decided to reference the local packages the linter started to warn.